pkg/apis/admission/install/roundtrip_test.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/testing/roundtrip"
+	admissionfuzzer "k8s.io/kubernetes/pkg/apis/admission/fuzzer"
+)
+
+func TestRoundTrip(t *testing.T) {
+	roundtrip.RoundTripTestForAPIGroup(t, Install, admissionfuzzer.Funcs)
```

Comment:
Why do we need a `roundtrip_test.go` in each group? Isn't the centralized [roundtrip](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/serialization_test.go#L206) test enough? We shouldn't distribute the burden of writing roundtrip tests to each API developer.

Breaking the monolithic fuzzer file makes sense.